.editorconfig

```diff
@@ -3,24 +3,15 @@ root = true
 [*]
 charset = utf-8
 end_of_line = lf
+indent_size = 4
 indent_style = tab
```

Comment:
Doesn't this mean 4 tabs per indent?

Reply:
It's how many columns an indent occupies; it'd mean 4 spaces, but still only 1 tab